src/components/Box.js

```diff
@@ -1,8 +1,14 @@
+import Later from './Later';
+import Joker from '../media/images/joker.jpg';
+//import Arrow from '../media/images/arrow.png';
+
 const Box = (props) => {
     return (
-        <div>
-            <div>{props.name}</div>
-            <div>{props.name2}</div>
+        <div className='article'>
+            {/* <div className='info'><img src={Arrow} alt='brak'></img></div> */}
+            <div className="pozniej"><Later link={props.link} name_article={props.name}/></div>
```

Comment:
propsy powinny by camelCase
```suggestion
            <div className="later"><Later link={link} nameArticle={name}/></div>
```